### DISCUSSION TITLE
Calendar in Dialog overlapping in Paradise Template

### DISCUSSION THREAD

**Discussion by ravikumar1243** [primefaces/community#699] [PrimeNG Templates]
In Paradise Template, If we use calendar (p-calendar) control inside dialog (p-dialog). When calendar expanded it is going behind the modal dialog. Could you suggest the fix.

**Comment by mertsincan** [comment]
Moved to PrimeNG Templates

**Comment by cetincakiroglu** [comment]
Could you please use appendTo property of the calendar https://primeng.org/calendar#api.calendar.props.appendTo
